pinot-query-planner/src/main/java/org/apache/pinot/query/planner/ExplainPlanStageVisitor.java

```diff
@@ -44,7 +44,7 @@
  */
 public class ExplainPlanStageVisitor implements StageNodeVisitor<StringBuilder, ExplainPlanStageVisitor.Context> {
 
-  private final QueryPlan _queryPlan;
```

Comment:
why are we changing explain plan stage visitor? this should not be used by distributed stage plan as it already doesn't contain the query plan

Reply:
distributed stage plan contains a local plan for that stage.

Reply:
context on Rong's comment: it's important to note that the distributed plan doesn't include the mailbox_receiver -> sender information, which makes the traversal "break". (note that `_sender` is `transient` under `MailboxReceiveNode`)

otherwise I agree that having this is helpful